src/javascript_bindings.pyx

```diff
@@ -100,10 +100,14 @@ cdef class JavascriptBindings:
             properties = self.properties
             objects = {}
             for objectName in self.objects:
-                methods = {}
-                for methodName in self.objects[objectName]:
-                    methods[methodName] = None
-                objects[objectName] = methods
+                attrs = {}
+                for name, value in self.objects[objectName].items():
+                    if self.IsValueAllowedRecursively(value) and \
+                            name == '__call__' or not name.startswith('_'):
```

Comment:
Um yeah this was a bit of a late workaround as I was getting a lot of extra builtin functions included in the mapping. My first reaction was to just exclude all private-like functions but now I'm not so sure that's necessary, it's adding an artificial limitation that's really not required.